frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion.unit.spec.tsx

```diff
@@ -59,7 +65,13 @@ jest.mock(
   () => VisualizationMock,
 );
 
-async function setup() {
+async function setup({
+  hash,
+  queryString,
```

Comment:
`queryString` isn't passed from tests, so you can hard-code it.

Reply:
you mean remove it?

Reply:
Sorry, I should've been more clear. I mean to remove it from the parameter and hard-code it where this param was used. Exactly like what you did.